packages/zero-runtime/README.md

```diff
@@ -23,6 +24,12 @@ A zero-runtime CSS-in-JS library that extracts the colocated styles to their own
 
 Zero-runtime supports Next.js and Vite with future support for more bundlers. You must install the corresponding plugin, as shown below.
 
+### Why this project exists?
+
+The initial motivator to work on this zero-runtime CSS-in-JS package was to improve Material UI's performance when it comes to styling. That's been a frequent pain point we've heard about ever since we introduced Emotion in v5. But another reason a zero-runtime styling solution is excellent for the future of Material UI is compatibility with React Server Components (RSCs).
```

Comment:
How about this (and keep in mind this is an initial draft of the Why statement; whenever we get close to a stable release, we should potentially expand):

> We first introduced CSS-in-JS to Material UI back in the v4 to v5 transition. We wanted to support color transformations, theme variables, and other features that wouldn’t be possible without a runtime solution. However, this has indeed impacted performance and has been a frequent feedback we’ve heard ever since.
> 
> With the recent CSS progress with features such as CSS variables and color-mix, to name a few, there’s no practical reason to keep a runtime solution. Additionally, React Server Components entered the mix of functionalities Material UI should support.
>
> We looked around and couldn’t find anything that fit precisely, so we started a new bespoke project. {newProductName} is designed to have a smooth migration experience from styled-components and Emotion.

Reply:
CSS-in-JS was introduced in Material UI v1, to support CSS media queries, hover, etc (we were coming from inline-style).

Reply:
Maybe this would be more accurate, then:

> We first introduced an Emotion-based CSS-in-JS styling engine to Material UI back in the v4 to v5 transition.